<launch>
  <node pkg="joy" type="joy_node" name="joy_node">
    <param name="autorepeat_rate" value="30" />
  </node>

  <node pkg="turtlejoy" name="turtlejoy_node" type="roscpp_node"/>

  <node pkg="floor_plane_ransac" type="floor_plane_ransac" name="floor_plane_ransac" output="screen">
      launch-prefix="xterm -e gdb --args"
      <param name="base_frame" value="/bubbleRob" />
      <param name="max_range" value="2.0" />
      <param name="n_samples" value="1000" />
      <param name="tolerance" value="0.05" />

      <remap from="/floor_plane_ransac/scans" to="/vrep/depthSensor"/>
  </node>

  <node pkg="rviz" name="rviz" type="rviz" />
</launch>
